applications/reference-data/src/main/java/no/dcat/themes/Controller.java

```diff
@@ -70,6 +70,21 @@
 
     }
 
+    @CrossOrigin
+    @RequestMapping(value = "/helptexts/", method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
```

Comment:
Her forventet jeg at det var mulig å få treff på /helptexts, men du krever at man skal bruke /helptexts/. Dette er ikke helt slik jeg tolker god RESTfull praksis. Er det noen grunn til det?

Reply:
Nei. Det er ingen grunn til det overhodet! :-) Det er nettopp det som er RESTful praksis. I REST er en URI et navn for en ressurs, intet mer, intet mindre. Det defineres her: http://www.ics.uci.edu/~fielding/pubs/dissertation/rest_arch_style.htm#tab_5_1 Altfor mange legger altfor mye annen betydning inn i det, men det er som oftest feil. Ofte ender man med å brekke [URI Opacity](https://www.w3.org/TR/webarch/#uri-opacity)

Det man kan gjøre, er å definere en indre struktur, man da må det gjøres in-band, men vi som server er uansett fri til å definere en hvilken som helst struktur.

Jo, forresten, det er en grunn, det er en del steder hvor det er konvensjon å bruke / for collections, som f.eks. i [Linked Data Platform](https://www.w3.org/TR/ldp/#ldpc) (som heller ikke er RESTful).

Men vi kan altså bruke hva som helst.